docs/docs/reference/docgen/javascript/auth/content/hasura-auth-client/content/02-sign-in.mdx

```diff
@@ -53,8 +53,8 @@ nhost.auth.signIn({ email: 'joe@example.com' })
 
 ```ts
 // [step 1/2] Passwordless sign in using SMS
-nhost.auth.signIn({ phoneNumber: '001122334455' })
+nhost.auth.signIn({ phoneNumber: '+11233213123' })
```

Comment:
Could you please add these changes to `packages/hasura-auth-js/src/hasura-auth-client.ts`? Our markdown files are automatically generated from JSDoc comments.

Reply:
yes will do that now. Sorry about that.

Reply:
@szilarddoro -- done!